docs/modding.md

```diff
@@ -78,3 +78,26 @@ See the example mod to learn about some of the properties that different types o
 //TODO help wanted!
 
 #### Items & Other Content
+
+Every item needs also their properties:
```

Comment:
You're not formatting the markdown list correctly, so it appears as a giant block of text. Start each line with a `-` and put a space after it so it looks like a list.